src/packagemanager.hpp

```diff
@@ -12,8 +12,8 @@ NOTE: Spaces are VERY important because we need them to append other strings to
 */
 
 struct PackageManager {
-	std::string search, install, uninstall, autoremove, update, upgrade, upgrade_pkg, clean;
+	std::string search, install, uninstall, autoremove, update, upgrade, upgrade_pkg, clean, layman;
```

Comment:
I would not add `layman` as an option supported for all package manager. The Gentoo packaging system is to my knowledge to only one that supports it. People could invoke it natively anyway.

Reply:
I just made the changes